docs/ide/reference/how-to-use-the-keyboard-exclusively.md

```diff
@@ -65,17 +65,17 @@ To move among the options in a dialog box and change option settings by using on
 
 ## Navigate between windows and files
 
-- To move among files in an editor or designer, choose the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Choose the **Enter** key to navigate to the highlighted file.
+- To move among files in an editor or designer, use the **Ctrl**+**Tab** keyboard shortcut to display the IDE Navigator with **Active Files** selected. Use the **Enter** key to navigate to the highlighted file.
 
-- To move among docked tool windows, choose the **Alt**+**F7** keyboard shortcut to display the IDE Navigator with **Active Tool Windows** selected. Choose the **Enter** key to navigate to the highlighted window.
+- To move among docked tool windows, use the **Alt**+**F7** keyboard shortcut to display the IDE Navigator with **Active Tool Windows** selected. Use the **Enter** key to navigate to the highlighted window.
```

Comment:
Same comment about "Use the Enter key" ... please change the verb "use" to "press" . Thanks!